packages/rocketchat-ui-account/client/accountProfile.js

```diff
@@ -56,16 +56,19 @@ Template.accountProfile.helpers({
 	},
 	services() {
 		const suggestions = Template.instance().suggestions.get();
-		return ['gravatar', 'facebook', 'google', 'github', 'gitlab', 'linkedIn', 'twitter']
-			.map((service) => {
+
+		if (suggestions.avatars) {
```

Comment:
Why remove the list of services? That will remove the suggestions for login to get the avatar

Reply:
This makes it use any provider instead of limiting to just these

Reply:
But them you remove the suggestions for login, is that expected?

Reply:
Ah... I see what you mean.  So this will get list of suggestions as well as show for them to link their account.  

So best way to handle is probably to add in the suggestions list to return even those that aren't already linked.

I'll take another look to see if can make that happen.

Reply:
@geekgonecrazy can you take another look at fixing this?